aim/sdk/objects/plotly.py

```diff
@@ -0,0 +1,33 @@
+from aim.sdk.num_utils import inst_has_typename
+from aim.storage.object import CustomObject
+from aim.storage.types import BLOB
+
+
+@CustomObject.alias("aim.plotly")
+class Plotly(CustomObject):
+    AIM_NAME = "aim.plotly"
```

Comment:
lets rename this to `aim.figure` or similar
in the future we might support more inputs (different from `plotly`)